src/engraving/libmscore/element.cpp

```diff
@@ -1368,22 +1377,22 @@ void paintElements(mu::draw::Painter& painter, const QList<Element*>& elements)
     QList<Ms::Element*> sortedElements = elements;
 
     std::sort(sortedElements.begin(), sortedElements.end(), [](Ms::Element* e1, Ms::Element* e2) {
-        if (e1->z() == e2->z()) {
-            if (e1->selected()) {
-                return false;
-            } else if (e1->visible()) {
-                return false;
-            } else if (e2->selected()) {
-                return true;
-            } else if (e1->visible()) {
-                return true;
-            }
+            if (e1->z() == e2->z()) {
+                if (e1->selected()) {
+                    return false;
+                } else if (e1->visible()) {
+                    return false;
+                } else if (e2->selected()) {
+                    return true;
+                } else if (e1->visible()) {
+                    return true;
+                }
 
-            return e1->track() > e2->track();
-        }
+                return e1->track() > e2->track();
+            }
 
-        return e1->z() < e2->z();
-    });
+            return e1->z() < e2->z();
+        });
```

Comment:
these were all perfectly indented before

Reply:
That is a problem with my Uncrustify file. (possibly caused by having an older version?)
I encounter the same problem in `uitypes.h` where i have to disable Uncrustify and manually reset it to whatever the codestyle approves. I'll do it here too :)

Reply:
This function has been removed in the latest master i believe

Reply:
A rebase would shoe, I guess 
It is needed anyhow

Reply:
and indeed element.cpp is one of the conflicting files